stage2/README.md

```diff
@@ -274,7 +289,7 @@ At the end of this week, all students will be required to use typescript and adh
   <summary markdown="span">Mentor activities</summary>
 
 - [Weekly meeting with students](https://docs.rs.school/#/mentoring?id=e%d0%b6%d0%b5%d0%bd%d0%b5%d0%b4%d0%b5%d0%bb%d1%8c%d0%bd%d1%8b%d0%b9-%d0%bc%d0%b8%d1%82%d0%b8%d0%bd%d0%b3-%d1%81%d0%be-%d1%81%d1%82%d1%83%d0%b4%d0%b5%d0%bd%d1%82%d0%b0%d0%bc%d0%b8)
-- [Checking the Draft version](https://docs.rs.school/#/pull-request-review-process) задания [Async Race](https://github.com/rolling-scopes-school/tasks/blob/master/tasks/async-race.md)
+- [Checking the Draft version](https://docs.rs.school/#/pull-request-review-process) задания [Async Race](./tasks/async-race/)
```

Comment:
```suggestion
- [Checking the Draft version](https://docs.rs.school/#/pull-request-review-process) of the task [Async Race](./tasks/async-race/)
```

Reply:
@KristiBo , thank you, I've already merged it, I'll fix it in the next PR.